libtiledbsoma/src/CMakeLists.txt

```diff
@@ -174,6 +174,10 @@ install(DIRECTORY
   ${CMAKE_CURRENT_SOURCE_DIR}/utils
   ${CMAKE_CURRENT_SOURCE_DIR}/external/include/span
   TYPE INCLUDE
+  FILES_MATCHING 
```

Comment:
It would be better to not install the directories but to only set the install targets to the specific files we want to include. This is how TileDB handles cmake installs for headers.